.github/ISSUE_TEMPLATE/new-article.md

```diff
@@ -0,0 +1,16 @@
+---
+name: New Article
+about: Pitch an idea for an article or claim one to write yourself.
+title: "Article: [Your title]"
+labels: "[Content] Article"
```

Comment:
Link "labels" to to [our repo's labels page](https://github.com/a11yproject/a11yproject.com/labels)? The idea is some users might not be familiar with working with them, so it could help show how we use them.

Reply:
This is actually in the front matter that tells GitHub what to do - it should automatically add this label to every new issue submitted with this template. Users won't see this section.

Reply:
Oh, I didn't know 😅 

Let's ignore this, then.